### PR TITLE
Modal is added which contains text and a close button.

### PR DESCRIPTION
**Pull request by vaishnavipaliya11**
Modal added with a close button .
image

### CODE REVIEW

**Review by saifudinbadani — APPROVED** [review]
The code looks clean and easy to understand. I think you have added css for grid component in this pr.

File: grids/grids.html [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+            style="
+              width: 682px;
+              height: 654px;
+              border: 0;
```
Comment (sushmita12345) [inline grids/grids.html:111]: For size greater than 10px use rem for responsiveness.

File: grids/grids.html [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+                  Rs 399 <span class="discount-para">(50% off)</span>
+                </p>
+                <a href="#" class="btn-card">Add to cart</a>
+              </div>
+            </div>
+          </div>
+          <div class="grid-element-two">
+            <div class="vertical-card-container">
+              <img class="card-img-top" src="/images/js.jpg" />
+              <div class="vertical-card-body">
```
Comment (sushmita12345) [inline grids/grids.html:94]: Suggest to use alt attribute in img tag.

File: grids/grids.html [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+            style="
+              width: 623px;
+              height: 878px;
```
Comment (sushmita12345) [inline grids/grids.html:168]: Here as well use rem instead of px.

File: modal/modal.css [hunk 4]
```diff
@@ -0,0 +1,22 @@
+.modal {
+    position: relative;
+    padding: 2rem 1rem;
+    border: 1px solid black;
+    border-radius: 0.35rem;
```
Comment (sushmita12345) [inline modal/modal.css:5]: for size less than 1 rem use px.

File: modal/modal.css [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+    border:1px solid black;
+    box-shadow: 5px 5px 20px #f5f5f5, -5px -5px 20px #ffffff;
+  }
+  .modal-btn {
+    font-size: small;
+    font-weight: bold;
+    border: 1px solid black;
+    border-radius:50%;
+    color: #262928e0;
+    margin: 1rem;
```
Comment (sushmita12345) [inline modal/modal.css:17]: use color variable

File: toast/toast.html [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+            style="
+              width: 682px;
+              height: 335px;
```
Comment (sushmita12345) [inline toast/toast.html:90]: Here as well use rem instead of px

**Review by sushmita12345 — APPROVED** [review]
There is some minor changes go with that and you can proceed to merge.

**Pull request merged**